blogs/whats_new_1.5/index.md

```diff
@@ -0,0 +1,178 @@
+
+## Feature Highlights
+
+Version 1.5 has significant improvements in ease-of-use and a number of features related to metrics.  The Flow Round Trip Time (RTT) feature that was in Technical Preview is now in General Availability (GA), which means it is fully supported.
+
+If you've used Network Observability before, the first thing you might have noticed after installing the operator is that there are two APIs available instead of one (Figure 1).
+
+![Network Observability Operator and APIs](images/noo-apis.png)
+_<div style="text-align: center">Figure 1: Network Observability Operater and APIs</div>_
+
+FlowMetrics is a dev preview feature which I will cover at the end, so let's start with FlowCollector.
+
+### FlowCollector API
+
+FlowCollector is the heart of network observability.  Creating a FlowCollector instance deploys an [eBPF](https://en.wikipedia.org/wiki/EBPF) agent for generating network flows, optionally supporting Kafka to improve scalability and reliability, a flowlogs pipeline for collecting, enriching, and storing the flow data and metrics, and a UI plugin for OpenShift web console to display graphs, tables, and topology.
+
+In 1.5, the FlowCollector API version was upgraded to *flows.netobserv.io/v1beta2* from v1beta1.  In web console, the UI or "Flow view" to create an instance gets a facelift.  The custom resource has the following top level categories:
+
+1. Name and Labels
+2. Loki client settings
+3. Console plugin configuration
+4. Namespace
+5. Deployment model
+6. Agent configuration
+7. Processor configuration
+
+The first significant change is in "Loki client settings".  When you click and open this, you get the following:
+
+![Loki client settings](images/loki_client_settings.png)
+_<div style="text-align: center">Figure 2: Loki client settings</div>_
+
+One of the new fields is "Mode" where you select how you installed Loki.  The most common is "LokiStack", which means you installed the Loki Operator and created a LokiStack instance.  Under the "Loki stack" section, make sure the **Name** matches the LokiStack name you gave it.  The nice part is that it will go and figure out the LokiStack gateway URL for you and give it proper authorization.
+
+Parameters are now exposed under "Console plugin configuration", particularly "Quick filters" (Figure 3).  Network Observability predefines some filters as defaults which can be changed.  While this was possible before, now you can do it in the UI.
+
+![Console plugin configuration](images/console_plugin_configuration.png)
+_<div style="text-align: center">Figure 3: Console plugin configuration</div>_
+
+Under "Agent configuration", there is no longer an agent type because the only supported agent is *eBPF*.  It is still possible to configure *IPFIX* through YAML, but support for OpenShift SDN as the Container Network Interface (CNI) which IPFIX requires, is also deprecated in OpenShift 4.15.
```

Comment:
ok I understand better what you said on slack :-)
Actually, OVN also uses OVS and can be configured with IPFIX, it's not only openshiftSDN

Reply:
Plus the IPFIX stuff is still available in the YAML so I would not mention it personnally